index.js

```diff
@@ -164,18 +164,23 @@ const getSettings = function(options) {
     excludeCoverage: ['test/**', path.join(__dirname, '**'), 'node_modules/**', 'package.json']
   };
 
+  const defaultBabel = {
+    babelrc: false,
+    exclude: path.join(process.cwd(), 'node_modules/**'),
+    presets: [
+      [presetEnv, {loose: true, modules: false, targets: {browsers: settings.browserslist}}]
+    ],
+    plugins: [
+      transformObjectAssign
+    ]
+  };
+
+  // get babel settings from the users provided function or use the defaults
+  settings.babel = options.babel ? options.babel(Object.assign({}, defaultBabel)) : defaultBabel;
```

Comment:
This shallow merge means that people can still mutate the `defaultBabel`. Maybe instead a new `defaultBabel` should be created for each call. For example:
```
const defaultBabel = () => ({
  // ...
});

settings.babel = options.babel ? options.babel(defaultBabel()) : defaultBabel;
```
I suppose this means the primed preset(s) or plugin(s) could be modified, but that seems less likely.